### ISSUE TITLE
trashbin_retention_obligation not respected in nextcloud 19

### ISSUE THREAD

**Issue by jknockaert** [nextcloud/server#22499]
I have in my configuration settings:
>'trashbin_retention_obligation' => '30,360'

After upgrading to nextcloud 19.0.2 all files older than 30 were deleted from the thrasbin by the cron script. Either the setting is not properly respected, or the definition of _enough free space_ changed. Before the upgrade files were deleted from the trashbin only after 360 days.

**Comment by acsfer** [comment]
Closing as duplicate of https://github.com/nextcloud/server/issues/22475

**Comment by jknockaert** [comment]
@acsfer Thanks for linking to the original report. I will add my observations there.
